src/Player.ts

```diff
@@ -148,7 +157,7 @@ export interface UIEvents {
   /**
    * Triggered when the hover position changes
    */
-  hoverPosition: { position: number; percent: number }
+  hoverPosition: HoverPosition | null
```

Comment:
Here we were using -1 for position/percent when not hovering which was messy, I'm changing this to `null` to simplify things.